evals/api.py

```diff
@@ -215,10 +234,51 @@ def sample_freeform(
         headers={},
         **kwargs,
     )
+    return postprocess_sample_freeform(
+        response,
+        actual_prompt,
+        metadata,
+        model_spec,
+        n_samples=n_samples,
+        return_logprobs=return_logprobs,
+        **kwargs)
+
+
+def postprocess_sample_freeform(
```

Comment:
Having an extra function here could be confusing. Is it possible to move the logic somewhere else? For example, could we could implement this inside CompletionFn instead?

Reply:
Was having similar thoughts: https://github.com/openai/evals/pull/515#discussion_r1156487521